homeassistant/helpers/aiohttp_client.py

```diff
@@ -93,6 +93,8 @@ def async_aiohttp_proxy_web(hass, request, web_coro, buffer_size=102400,
     yield from async_aiohttp_proxy_stream(hass, request, req.content,
                                           req.headers.get(CONTENT_TYPE))
 
+    req.close()
```

Comment:
use:
```
try:
   yield from async_aiohttp_proxy...
finally:
  req.close()
```

Reply:
Doesn't aiohttp do this automatically? https://github.com/aio-libs/aiohttp/blob/bb646d143cc7bd3b1a614d7c101957b56b11a67f/aiohttp/client_reqrep.py#L500